myria/cluster/scripts/cli.py

```diff
@@ -1141,7 +1141,20 @@ def run():
     help="Number of Myria workers per cluster node [default: %d]" % INSTANCE_TYPE_CONFIGS[DEFAULTS['instance_type']].workers_per_node)
 @click.option('--cluster-log-level', show_default=True,
     type=click.Choice(LOG_LEVELS), default=DEFAULTS['cluster_log_level'])
+@click.option('--perfenforce', is_flag=True, help="Enable PerfEnforce (will override default cluster configuration)")
 def create_cluster(cluster_name, **kwargs):
+    # If perfenforce is enabled, override the cluster configuration
+    if kwargs['perfenforce']:
+        click.echo("Adjusting cluster options for PerfEnforce...")
```

Comment:
Do you want to allow users to override any of the default options for PerfEnforce (say, to pick a different instance type)? If not, then we need to add a callback to trigger an error if any of these options are specified when the `--perfenforce` option is present.